$NetBSD: patch-configure,v 1.1 2021/01/16 14:54:14 mef Exp $

https://openbsdmailbox.blogspot.com/2020/12/graphicspdf2djvu-unbreak-for-poppler.html
the pdf2djvu configure script performs checks against poppler-splash,
but poppler-splash.pc had been removed in poppler-20.12. Check against
poppler itself to unbreak the build. 

--- configure.orig	2020-08-08 02:13:47.000000000 +0900
+++ configure	2021-01-16 23:46:31.375535210 +0900
@@ -4006,19 +4006,19 @@ $as_echo "ok" >&6; }
 
 
 pkg_failed=no
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for poppler-splash >= 0.35.0" >&5
-$as_echo_n "checking for poppler-splash >= 0.35.0... " >&6; }
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for poppler >= 0.35.0" >&5
+$as_echo_n "checking for poppler >= 0.35.0... " >&6; }
 
 if test -n "$POPPLER_CFLAGS"; then
     pkg_cv_POPPLER_CFLAGS="$POPPLER_CFLAGS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"poppler-splash >= 0.35.0\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "poppler-splash >= 0.35.0") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"poppler >= 0.35.0\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "poppler >= 0.35.0") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_POPPLER_CFLAGS=`$PKG_CONFIG --cflags "poppler-splash >= 0.35.0" 2>/dev/null`
+  pkg_cv_POPPLER_CFLAGS=`$PKG_CONFIG --cflags "poppler >= 0.35.0" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -4030,12 +4030,12 @@ if test -n "$POPPLER_LIBS"; then
     pkg_cv_POPPLER_LIBS="$POPPLER_LIBS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"poppler-splash >= 0.35.0\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "poppler-splash >= 0.35.0") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"poppler >= 0.35.0\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "poppler >= 0.35.0") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_POPPLER_LIBS=`$PKG_CONFIG --libs "poppler-splash >= 0.35.0" 2>/dev/null`
+  pkg_cv_POPPLER_LIBS=`$PKG_CONFIG --libs "poppler >= 0.35.0" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -4056,14 +4056,14 @@ else
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        POPPLER_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "poppler-splash >= 0.35.0" 2>&1`
+	        POPPLER_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "poppler >= 0.35.0" 2>&1`
         else
-	        POPPLER_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "poppler-splash >= 0.35.0" 2>&1`
+	        POPPLER_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "poppler >= 0.35.0" 2>&1`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$POPPLER_PKG_ERRORS" >&5
 
-	as_fn_error $? "Package requirements (poppler-splash >= 0.35.0) were not met:
+	as_fn_error $? "Package requirements (poppler >= 0.35.0) were not met:
 
 $POPPLER_PKG_ERRORS
 
@@ -4095,7 +4095,7 @@ else
 $as_echo "yes" >&6; }
 
 fi
-poppler_version=$($PKG_CONFIG --modversion poppler-splash)
+poppler_version=$($PKG_CONFIG --modversion poppler)
 
 cat >>confdefs.h <<_ACEOF
 #define POPPLER_VERSION_STRING "$poppler_version"
